src/entities/memoryPracticeQuestion.entity.ts

```diff
@@ -14,7 +14,7 @@ import { User } from './user.entity';
 
 @Entity({ name: 'MemoryPracticeQuestion' })
 export class MemoryPracticeQuestion {
-  @PrimaryColumn()
+  @Column()
```

Comment:
the 'uid' field is already defined in this line
```
@JoinColumn({ name: 'uid', referencedColumnName: 'uid' })
  users: User;
```
so I think the following code can be removed
```
@Column()
  uid: number;
```

Reply:
Not sure why but I need this to query using the uid... has this never been a problem for the other files?

Reply:
what you did is already correct. sorry for the confusion caused naa